### ISSUE TITLE
HA for cassandra with API Gateway

### ISSUE THREAD

**Issue by johnfelipe** [TrivadisPF/platys-modern-data-platform#532]

**Comment by johnfelipe** [comment]
is possible?

**Comment by johnfelipe** [comment]
Anyone?

**Comment by gschmutz** [comment]
@johnfelipe Don't understand what an API gateway would bring to Cassandra for HA. Cassandra will be HA, if it runs as a cluster, which is supported by this stack. But it is just for demonstration reasons, as a cluster on a single machine (docker-compose just runs on one machine) does not bring any value for HA.

**Comment by johnfelipe** [comment]
> @johnfelipe Don't understand what an API gateway would bring to Cassandra for HA. Cassandra will be HA, if it runs as a cluster, which is supported by this stack. But it is just for demonstration reasons, as a cluster on a single machine (docker-compose just runs on one machine) does not bring any value for HA.

That is true, is only testing about HA using API gateway, can u elaborate and example for Cassandra and API gateway??
